### DISCUSSION TITLE
Transcription Edit - it is keeping on skipping pages while editing transcript

### DISCUSSION THREAD

**Discussion by EducateOne** [AI4Bharat/Chitralekha#12] [Q&A]
As I edit subtitles on any page, at the end of the page it automatically skips many pages. E.g. I was editing page 5 using the "transcript edit" task, as I finish, it jumped to page 18 by skipping in-between pages. And I am not able to delete the "repetitive and redundant text" from the in-between skipped pages, as it regenerate during the submission, and "repetitive and redundant text" appears in the output files. I hope someone has solution for this problem. Please help.

**Comment by aravinth** [comment]
Hi @EducateOne - Can you please share your organization & the task ID ? We will try to debug from backend.

**Reply by EducateOne** [reply]
Thank you for your help. Organization: **NPTEL Org**, Task ID: **9803**.

**Reply by EducateOne** [reply]
Hi @aravinth, I had communicated with NPTEL Org, they have fixed the problem now.
But please can you tell me: If I press "ctrl+R" while editing the transcript, does it affect the previously saved changes? Because I didn't try this, as the pop-up massage says, "Reload site? Changes you made may not be saved." 
TIA.

**Reply by Shruti1229** [reply]
Some points to be noted:
1. You need to do ctr+r, only the first time, you land on an "Edit" page from tasks page.
2. Do not work on multiple tabs.
3. If you are working on multiple systems, make sure to logout from the tool, every time you use it.
4. Auto-save is called every minute. But, before you move to the next page, press on "Save" button explicitly.

**Reply by EducateOne** [reply]
Thank you for your support. I followed all things as you mentioned. But, again the problem was reappeared. Now I am getting multiple overlapped subtitles on single timeline. Please can you tell why it is happening.
